README.md

```diff
@@ -1690,8 +1690,28 @@ Forked from [AirBNB's style guide](https://github.com/airbnb/javascript)
       name: 'yup',
     });
     ```
+  - [22.4](#22.4) <a name='22.4'></a> Use PascalCase or camelCase for acronyms in names.
+    ```javascript
+    // bad
+    const baseURL = 'https://airbnb.com';
```

Comment:
Could be -- originally was gonna do that, but wanted to leave this open in case we wanted to try and contribute back to the AirBnb guide!